dotnet-authserver/src/TeacherIdentity.AuthServer/Journeys/TrnTokenHelper.cs

```diff
@@ -138,24 +134,47 @@ public async Task InvalidateTrnToken(string trnTokenValue, Guid userId)
         };
     }
 
-    private async Task<User> UpdateUserTrn(User user, string trn)
+    private async Task<User> UpdateUserFromToken(User user, string trn)
     {
-        user.Trn = trn;
-        user.TrnLookupStatus = TrnLookupStatus.Found;
-        user.TrnAssociationSource = TrnAssociationSource.TrnToken;
-        user.Updated = _clock.UtcNow;
+        var dqtUser = await _dqtApiClient.GetTeacherByTrn(trn);
```

Comment:
We may be able to avoid this additional call to dqt as we make a call at the start of the journey when we first get the token. It would require a larger refactor though